content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -150,6 +151,7 @@ different Kubernetes components.
 | `ServiceTopology` | `false` | Alpha | 1.17 | |
 | `SetHostnameAsFQDN` | `false` | Alpha | 1.19 | 1.19 |
 | `SetHostnameAsFQDN` | `true` | Beta | 1.20 | |
+| `StorageVersionAPI` | `false` | Alpha | 1.20 | |
```

Comment:
Is this intended? The PR title doesn't mention `StorageVersionAPI`.

If I were documenting 2 different features I'd probably do that in 2 separate commits - it's easier to revert if ever needed.

Reply:
Yes there's two separate things, storage version depends on the identity flag being on. @roycaihw is there a second PR documenting storage version? Should these be moved there?

Reply:
Storage version API is not user-facing. The only user of it is the [storage migrator](https://github.com/kubernetes-sigs/kube-storage-version-migrator). Currently there is no plan to have a k8s.io doc for it. We may document it in the storage migrator repo. cc @caesarxuchao 

I removed this line. 

(we have the k8s.io doc for apiserver-identity because it can be used for other features, e.g. priority & fairness)